assets/css/src/components-public/_resets.scss

```diff
@@ -0,0 +1,104 @@
+// basic default styling
+// to allow for consistent styling between mailpoet plugin and user's theme
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+.mailpoet_form {
+  body {
+    color: rgb(40, 48, 61);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: normal;
+    text-align: left;
+  }
+
+  html,
+  body,
```

Comment:
How `html` and `body` tags can be inside a `.mailpoet_form` class? This class is applied to a `div` inside the iframe `body`, as far as I can see. At the same time, we don't want to affect elements which don't belong to MailPoet, see my [other comment](https://github.com/mailpoet/mailpoet/pull/3798/files#r752046427).